textutils_internal.h

```diff
@@ -197,51 +197,43 @@ const char *hts_strprint(char *buf, size_t buflen, char quote,
 // Faster replacements for strtol, for use when parsing lots of numbers.
 // Note that these only handle base 10 and do not skip leading whitespace
 
-/// Convert a string to a signed integer, with overflow detection
```

Comment:
Is there a good reason why these functions swapped round?  It makes spotting the differences much harder than if they stayed in the original order.

Reply:
There was, but it's no longer relevant so I can swap them back.

Originally I implemented the signed conversion as a call out to the unsigned one for the non-negative case.  This had a huge speed increase.  However with hindsight that was purely because the few extra bytes of code shuffled subsequent functions down and changed the behaviour of the instruction case alignment.  Once I realised that was the cause of the wild differences, the other changes were mostly undone.